spec/requests/api/v1/users/create_spec.rb

```diff
@@ -5,19 +5,16 @@
     post api_v1_users_path, params: params, as: :json
   end
 
-  let(:first_name) { 'Obi Wan' }
-  let(:last_name) { 'Kenobi' }
   let(:email) { 'obikenobi@rebel.com' }
   let(:password) { 'abcd1234' }
-  let(:locale) { 'es' }
   let(:params) do
     {
       user: {
-        first_name: first_name,
-        last_name: last_name,
+        first_name: 'Obi Wan',
```

Comment:
why does this change doesn't apply for all params?

Reply:
Those attributes are the same for all tests, the other params are being changed in the next tests to modify the request
![Screen Shot 2020-10-19 at 11 36 02](https://user-images.githubusercontent.com/33032571/96465425-55474980-11ff-11eb-8ac7-e650cb38ef08.png)